cssdb.settings.json

```
@@ -983,6 +983,25 @@
       }
     ]
   },
+  {
+    "id": "unset-value",
+    "title": "`unset` Keyword",
+    "description": "The unset CSS keyword resets a property to its inherited value if the property naturally inherits from its parent, and to its initial value if not.",
+    "specification": "https://www.w3.org/TR/css-cascade-4/#inherit-initial",
+    "stage": 3,
+    "mdn_path": "css.types.global_keywords.unset",
+    "browser_support": {},
+    "docs": {
+      "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/unset"
+    },
+    "example": "div {\n  border-color: unset;\n  color: unset;\n}",
+    "polyfills": [
+      {
+        "type": "PostCSS Plugin",
+        "link": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-unset-value"
+      }
+    ]
```

```suggestion
    "example": "div {\n  border-color: unset;\n  color: unset;\n}"
```

Not released yet, but wanted it visible somewhere on github